### DISCUSSION TITLE
Question about the results of the DepthMap node

### DISCUSSION THREAD

**Discussion by TheMoye** [alicevision/Meshroom#2457] [Q&A]
Hello everyone,

I have been trying to create a 3D model of a head using Meshroom for a few days now, to then 3D print it, but I'm struggling to get good results. There is always a lot of noise, making the face look like it is covered in bumps and dents. In short, I'm exploring the parameters of the different nodes.
And there is one node that intrigues me: the "DepthMap" node (and its cousin "DepthMapFiltering"). Coming from the world of 3D modeling, I have a precise idea of what a depth map should look like.
But here is what Meshroom gives me:

![depthMap](https://github.com/alicevision/Meshroom/assets/46683101/797d7562-8f7f-42d4-b711-4827713ecd02)

Is this normal? Is it a misconfiguration on my part?
Thanks in advance for your help.

**Comment by natowi** [comment]
The DepthMap will look as in your image when opened in an image viewer. You an preview the DepthMap in the 3D Viewer using this button: 
![grafik](https://github.com/alicevision/Meshroom/assets/8138537/95457281-3f9c-4cf5-86d0-0f025fbb6720)

**Comment by TheMoye** [comment]
Hello @natowi , and thank you for this response,

I didn't know about this button! But unfortunately, when I click on it, Meshroom crashes and closes 🤷

**Comment by TesseractCat** [comment]
I'm also having the same issue with my depthmaps. The aforementioned button causes Meshroom to crash for me as well. Any updates?
